tianshou/policy/imitation/base.py

```diff
@@ -36,6 +37,12 @@ class ImitationPolicy(BasePolicy):
         explanation.
     """
 
+    @dataclass(kw_only=True)
+    class LossStats(BaseStats):
```

Comment:
I think it's better to move all the `LossStats` classes out of the Policy class to make it more explicit instead of a nested one.